packages/checkout/widgets-lib/src/widgets/sale/views/PaymentMethods.tsx

```diff
@@ -82,7 +83,12 @@ export function PaymentMethods() {
     }
   }, [paymentMethod]);
 
-  useEffect(() => sendPageView(SaleWidgetViews.PAYMENT_METHODS), []); // checkoutPrimarySalePaymentMethodsViewed
+  useEffect(() => {
+    // We need OrderQuote on this page view, but don't want to slow load of page by waiting for it
+    if (!orderQuote || orderQuote?.totalAmount?.USD?.amount === undefined) return;
```

Comment:
Given that USD is not guarantee to exist, and we don't know yet the settlement coin the user will select.
We may do better by sending a serialised `orderQuote: totalAmount` thru.

In most cases should be just like this, IMO:
`{ pricing: { USDC: x, GOG: x, ETH: x, USD: x }}`